package.json

```diff
@@ -12,8 +12,16 @@
 	"bin": {
 		"chalk": "cli.js"
 	},
+	"type": "module",
 	"engines": {
-		"node": ">=6"
+		"node": "^12.20.0 || ^14.13.1 || >=16.0.0"
+	},
+	"xo": {
+		"rules": {
+			"comma-dangle": ["warn", "only-multiline"],
+			"promise/prefer-await-to-then": "warn",
+			"arrow-body-style": "warn"
```

Comment:
Not sure what @sindresorhus's stance is but I don't like warnings these days, I generally prefer errors or nothing at all.

Thoughts?

Reply:
The rules should be followed instead of ignored.

Reply:
So then these should be `"error"`.

Reply:
Ah okay! Yeah I'd rather follow the rules than use warnings as well. The warnings were because I started working on this before #16 was merged and I was trying not to make too many changes that could cause conflicts. Now that #16 is merged, that is no longer a concern and I can change this to a more pure approach.

Reply:
Done. Changed the warnings to errors. Now there are 2 errors. Before fixing the errors, I'll quickly point out that `xo` and GitHub Actions `CI are doing their jobs to give us feedback to improve the code.

![Screen Shot 2021-09-14 at 8 43 03 AM](https://user-images.githubusercontent.com/305268/133291356-2b586c1a-2068-492e-b6c7-f06cf58f9008.png)

![Screen Shot 2021-09-14 at 8 43 29 AM](https://user-images.githubusercontent.com/305268/133291756-1bd784c5-4d52-4b6f-8f46-6cfde7896943.png)